data/json/recipes/recipe_others.json

```diff
@@ -4535,5 +4535,29 @@
       [ [ "hose", 1 ] ],
       [ [ "condom", 1 ], [ "balloon", 1 ] ]
     ]
+  },
+  {
+    "type": "recipe",
+    "result": "tempered_glass_sheet",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "fabrication",
+    "difficulty": 4,
+    "time": "1 h",
+    "autolearn": true,
+    "tools": [ [ [ "tongs", -1 ] ], [ [ "crucible", -1 ], [ "crucible_clay", -1 ] ], [ [ "sheet_metal", -1 ] ], [ [ "forge", 75 ] ] ],
+    "components": [ [ [ "glass_shard", 400 ], [ "flask_glass", 126 ], [ "jar_glass", 42 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "tempered_glass_sheet",
```

Comment:
Duplicated recipe here (identical to the recipe above), probably should be removed.

Reply:
ill get round to finishing this soon and fixing some things